api/main.py

```
@@ -6,6 +6,8 @@
 from PIL import Image  
 from io import BytesIO
 import base64
+import os
+from dotenv import load_dotenv
```

Ruff (0.8.2)

`dotenv.load_dotenv` imported but unused

Remove unused import: `dotenv.load_dotenv`

(F401)